app/sessions/sessions_manager.go

```diff
@@ -69,6 +75,30 @@ func (sm *SessionsManager) initWatchers() {
 	go sm.consumeInstanceStopped(instanceStopped)
 }
 
+func (sm *SessionsManager) initContainerManagementStart() {
+	output, err := sm.provider.GetManagedContainers()
+
+	if err != nil {
+		log.Debugf("Could not get list of managed containers to apply default")
+	}
+
+	var startContainers []string
+
+	for _, name := range output {
+		instance, err := sm.provider.GetState(name)
+
+		if err != nil {
+			log.Debugf("Could not get state for %s", name)
+		}
+
+		if instance.IsReady() {
+			startContainers = append(startContainers, name)
+		}
+	}
+
+	sm.RequestSession(startContainers, sm.config.Sessions.DefaultDuration)
```

Comment:
I'm not really sure about this. I believe the behavior is that we want to **stop** the instances started we found which are **not** in the KV.

Reply:
I disagree with this partially. I believe that if I have an already running containers, and I restart sablier, it would bring my instances down instantly. I prefer having it this way, but I also see your argument for instantly bringing them down.

Maybe we can create a setting that is like STARTUP_DEFAULT_EXPIRATION_DURATION which can be set to "0m" for instant or whatever the user sets. If the setting it blank it defaults to the SESSIONS_DEFAULT_DURATION.

Would that work for you?